Steps/PrGate.yml

```diff
@@ -42,14 +42,18 @@ parameters:
   displayName: Perform Stuart PR Evaluation
   type: boolean
   default: true
-- name: tool_chain_tag
-  displayName: Tool Chain (e.g. VS2022)
-  type: string
-  default: ''
+- name: extra_install_step
+  displayName: Extra Install Steps
+  type: stepList
+  default: []
 - name: install_tools
   displayName: Install Build Tools
   type: boolean
   default: true
+- name: tool_chain_tag

```

Comment:
Note: The `install_tools` parameter is just moved above `tool_chain_tag` here to keep things in ascending order.